docs/remote/devcontainerjson-reference.md

```diff
@@ -50,6 +50,8 @@ While some devcontainer.json properties apply generally, others are only used in
 | `extensions` | array | An array of extension IDs that specify the extensions that should be installed inside the container when it is created. Defaults to `[]`. |
 | `settings` | object | Adds default `settings.json` values into a container/machine specific settings file.  |
 | `forwardPorts` | array | An array of ports that should be forwarded from inside the container to the local machine. |
+| `portsAttributes` | object | Sets default properties for specific forwarded ports. Can use a port number or a regular expression. |
```

Comment:
It can also be a port range.

Reply:
Thanks for catching this! Added "range" to the description.